aten/src/ATen/templates/NativeFunctions.h

```diff
@@ -31,6 +31,15 @@ inline Tensor from_blob(
   return options.type().tensorFromBlob(data, sizes, deleter);
 }
 
+inline Tensor from_blob(
```

Comment:
Where do these blobs usually come from?

Reply:
It's for any external pointer that you want to wrap into a tensor. @goldsborough another one that needs doc'ing.

Reply:
But then why isn't this a constructor of Tensor?

Reply:
this name already exists (see above in the file).